src/modules/claim/claim.dto.ts

```diff
@@ -2,13 +2,15 @@ import { IClaimIssuance, IClaimRejection, IClaimRequest } from './claim.types';
 import {
   IsArray,
   IsBoolean,
+  IsNumberString,
   IsString,
   validateOrReject,
 } from 'class-validator';
 import { ApiProperty } from '@nestjs/swagger';
 
 export class ClaimRequestDTO implements IClaimRequest {
   static async create(data: Partial<ClaimRequestDTO>) {
+    data.claimTypeVersion = data.claimTypeVersion.toString().split('.')[0];
```

Comment:
Wouldn't this lead to duplicate enrollment requests? If versions in the role definition and in the claim are different, how  to determine that enrollment request already submitted?
Also `claimTypeVersion` in iam-client-lib just defined as number, but it sends to cache-server whatever it receives from switchboard-dapp

Reply:
I don't think this really impacts duplicate enrolment requests as long as the check on the frontend (which is the only place that we have the check currently) is correct. I'm making a change to that here: https://github.com/energywebfoundation/switchboard-dapp/pull/293